src/main/java/Duke.java

```diff
@@ -1,10 +1,55 @@
+import commands.Command;
+import exceptions.DukeException;
+import utils.Parser;
+import utils.Storage;
+import utils.TaskList;
+import utils.Ui;
+
 public class Duke {
+    private Storage storage;
+    private Ui ui;
+    private TaskList tasks;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke duke = new Duke("/home/abhinav/Desktop/2103T/duke/data/dukeTest.txt");
```

Comment:
Sorry to nitpick but I have a minor gripe with the file path. I would recommend using a relative file path to create the 'dukeTest.txt' as if you send the jar executable to other users, they would not have the '/home/abhinav' directory and having the program create a brand new set of directories there would not be that good an idea.